erpnext/crm/doctype/sales_stage/sales_stage.json

```diff
@@ -33,10 +36,20 @@
    "role": "Sales Manager",
    "share": 1,
    "write": 1
+  },
+  {
+   "email": 1,
+   "export": 1,
+   "print": 1,
+   "report": 1,
+   "role": "Sales User",
+   "select": 1,
```

Comment:
Please add "Read" permission for Sales Stage as well
Add "Read" and "Select" permission for "Sales User (Ready Only)" as well